extensions/cornerstone-dicom-sr/src/utils/addMeasurement.ts

```diff
@@ -45,6 +45,13 @@ export default function addMeasurement(
 
   const annotationManager = annotation.state.getDefaultAnnotationManager();
 
+  // Create Cornerstone3D Annotation from measurement
+  const frameNumber =
+    (measurement.coords[0].ReferencedSOPSequence &&
+      measurement.coords[0].ReferencedSOPSequence[0] &&
+      measurement.coords[0].ReferencedSOPSequence[0].ReferencedFrameNumber) ||
```

Comment:
use optional chanining https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining

Reply:
fixed in a9d22efe494178b6cbbdaaf971700f4b9e1bf517